### PR TITLE
Implement liking and unliking block producer

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +111,34 @@ $ curl -X POST -d '{"old_password":"dmytro.striletskyi.1337", "new_password":"dm
...
+| :-----------------------: | :-----: | :------: | ----------------------------- |
+| block_producer_identifier | Integer | Yes      | Identifier of block producer. |
+
+```bash
+$ curl -X POST \
+      -H "Content-Type: application/json" \
+      -H "Authorization: JWT eyJ0e....eyJ1c2VyX2....sOx4S9zpC..." \
+      http://localhost:8000/block-producers/2/like/ | python -m json.tool
+{
+    "message": "User has been created.",
```
Comment (anastasiia-bilova) [inline README.md]: Update output result.

File: README.md [hunk 2]
```diff
@@ -110,6 +111,34 @@ $ curl -X POST -d '{"old_password":"dmytro.striletskyi.1337", "new_password":"dm
...
+    "status_code": 200
+}
+```
+
+##### Known errors
+
+| Argument     | Level                      | Error message                                                    | Status code |
```
Comment (anastasiia-bilova) [inline README.md]:
```suggestion
| Argument     | Level           | Error message                                                    | Status code |
```

File: README.md [hunk 3]
```diff
@@ -110,6 +111,34 @@ $ curl -X POST -d '{"old_password":"dmytro.striletskyi.1337", "new_password":"dm
...
+| :----------: | :------------------------: | ---------------------------------------------------------------- | :---------: |
+| -            | General execution          | User with specified e-mail address does not exist.               | 400         |
+| -            | General execution          | Block producer with specified identifier address does not exist. | 400         |
```
Comment (anastasiia-bilova) [inline README.md]:
```suggestion
| -            | General execution          | Block producer with specified identifier does not exist. | 400         |
```

File: directory/block_producer/domain/objects.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+"""
+Provide implementation of user registration.
```
Comment (anastasiia-bilova) [inline directory/block_producer/domain/objects.py]: Update documentation.

File: directory/block_producer/domain/objects.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        """
+        Constructor.
+        """
+        self.user = user
+        self.block_producer = block_producer
+        self.block_producer_like = block_producer_like
+
+    def do(self, user_email, block_producer_id):
+        """
+        Like a block producer.
```
Comment (anastasiia-bilova) [inline directory/block_producer/domain/objects.py]:
```suggestion
        To like a block producer.
```

File: directory/block_producer/tests/views/test_like.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
+"""
+Provide tests for implementation of single user password endpoint.
```
Comment (anastasiia-bilova) [inline directory/block_producer/tests/views/test_like.py]:
```suggestion
Provide tests for implementation of single block producer like endpoint.
```

File: README.md (hunk 1 above)
Comment (dmytrostriletskyi) [reply README.md]: @anastasiia-bilova, done.

File: README.md (hunk 2 above)
Comment (dmytrostriletskyi) [reply README.md]: @anastasiia-bilova, done.

File: README.md (hunk 3 above)
Comment (dmytrostriletskyi) [reply README.md]: @anastasiia-bilova, done.

File: directory/block_producer/domain/objects.py (hunk 4 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/domain/objects.py]: @anastasiia-bilova, done.

File: directory/block_producer/domain/objects.py (hunk 5 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/domain/objects.py]: @anastasiia-bilova, done.

File: directory/block_producer/tests/views/test_like.py (hunk 6 above)
Comment (dmytrostriletskyi) [reply directory/block_producer/tests/views/test_like.py]: @anastasiia-bilova, done.

**Pull request merged**